core-services/ci-search-api-ci/api-ci-deploy.yaml

```diff
@@ -0,0 +1,24 @@
+# This file contains the Ci search pieces that are specific to the api.ci cluster
```

Comment:
I feel like we've still not really solved the `core-services/` versus `clusters/` dilemma. Seems weird to have to call this out as `api-ci` in the dir name?

Not sure we need to change it here but we should pick a reasonable workflow and run with it.

Reply:
I'm all for moving `core-services/*` (and `services`) under `clusters/api.ci`, it's just a real PITA to do (especially with items tracked by config updater). But we'll need to bite the bullet eventually.

Reply:
Yeah the exception to that rule is IMHO prow config, because its changed not only by us but also by others, so not moving it around would make things easier.